OpenEdXMobile/src/main/java/org/edx/mobile/view/OfflineSupportBaseActivity.java

```diff
@@ -19,16 +22,19 @@
  * {@link SnackbarErrorNotification} should never appear until and unless the
  * {@link FullScreenErrorNotification} is hidden.
  */
-public abstract class OfflineSupportBaseActivity extends BaseSingleFragmentActivity
+public abstract class OfflineSupportBaseActivity<VB extends ViewDataBinding> extends BaseFragmentActivity
         implements SnackbarStatusListener, RefreshListener {
 
+    protected VB binding;
+
     private SnackbarErrorNotification snackbarErrorNotification;
     private boolean isFullScreenErrorVisible = true;
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
-        snackbarErrorNotification = new SnackbarErrorNotification(findViewById(R.id.coordinator_layout));
+        binding = DataBindingUtil.setContentView(this, R.layout.activity_main_dashboard);
```

Comment:
Please check this 😛 
either use a dynamic approach or custom view binding for a specific view

Reply:
My bad. 🤦🏻  Let me correct it.